src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/AzureAttestationBasedEnclaveProvider.cs

```diff
@@ -73,15 +73,14 @@ internal override void GetEnclaveSession(string servername, string attestationUr
         // Gets the information that SqlClient subsequently uses to initiate the process of attesting the enclave and to establish a secure session with the enclave.
         internal override SqlEnclaveAttestationParameters GetAttestationParameters(string attestationUrl, byte[] customData, int customDataLength)
         {
-            ECDiffieHellmanCng clientDHKey = new ECDiffieHellmanCng(DiffieHellmanKeySize);
-            clientDHKey.KeyDerivationFunction = ECDiffieHellmanKeyDerivationFunction.Hash;
-            clientDHKey.HashAlgorithm = CngAlgorithm.Sha256;
+            ECDiffieHellman clientDHKey = ECDiffieHellman.Create();
```

Comment:
Do the derivation function and algorithm no longer need to be set because the values required are the defaults? If so it might be worth putting a comment in about that.

Reply:
Those values are specified when calling `DeriveKeyFromHash`. Added clarifying comments.